### ISSUE TITLE
Website Architecture Changes

### ISSUE THREAD

**Issue by camposcristian** [aimementoring/website#4]
@kbardi @lulen11 feel free to add/edit/remove anything on this list if needed to make sure the Epic is updated!

## High Priority Tasks before the Deploy on Jan 21st 🚀
- [x] Merge `staging` to `master` and test how the budget for lighthouse works @kbardi @bronz3beard
- [x] Loading Process Research and Implementation  (Remove Loaders / Add Placeholders for images and CSS page scaffolding, primarily `header`, `footer` and `logos` components) **3d** @kbardi 
- [x] Make sure we test [Be a Mentor](http://v5.aimementoring.com/be-a-mentor), [Donate](http://v5.aimementoring.com/donate) and [Positions](http://v5.aimementoring.com/positions) with at least one submission each 
- [x] Modify bugsnag to go to this new architecture

## Nice to have
- [ ] Get[ inline critical CSS replacing the basic CSS](https://github.com/zeit/next-plugins/issues/238#issuecomment-432211871) so is loaded only once @kbardi 
- [ ] Close the PR of Apparel and transfer the structural work we did @davidpaley 
- [ ] Remove Craft completely
- [ ] Use [Github Secrets](https://help.github.com/en/articles/virtual-environments-for-github-actions) for any tokens we need and look for a way to share them on Heroku if needed

## Done
- [x] Implement SEO for user browser and bots @davidpaley 
- [x] Transfer the issues left to the new repo @mischacolley 
- [x] Redesign Be a Mentor to only include a login (@lulen11 @mischacolley maybe get a quick redesign without the form?) 
- [x] Fix the carousel in /stories page @davidpaley 
- [x] Transfer Be a Mentor @davidpaley 
- [x] ❗️**High Priority!** Fix assets deployment and compress them using gzip in Cloudfront  @kbardi 
- [x] Research SEO and the content model on CMSs to include images @mischacolley 
- [x] Lazy load using [@loadable/component](https://github.com/smooth-code/loadable-components) instead of Loadable @kbardi 
- [x] Check and fix env variables @davidpaley 
- [x] Migrate contact forms to Intercom @bronz3beard #7 
- [x] Transfer Donate @davidpaley 
- [x] Transfer Positions to this new architecture @bronz3beard 
- [x] Remove static.json . Look for alternatives using Contentful  #12 @bronz3beard 
- [x] Start using CloudFront CDN for assets
- [x] Deploy the website assets (website+blueprint js+css) and access from a CDN @camposcristianeze 1d //I'll get this one on Friday.
- [x] Migrate pages [left behind](https://docs.google.com/spreadsheets/d/1zFOjBOt2IJFkjvrWxf-OHwQVNKWPH-YjtumqUuG9nvE/edit#gid=0) @kbardi **1,5d**
- [x] Rename `website` for `website-v4` and `website-ssr-blueprint-poc` for `website` @camposcristianeze 
- [x] Transfer latest changes from the past 3 weeks @kbardi 
- [x] Test inline vs chunk JS in S3 for the critical path and add this automatically to the index.html @camposcristianeze 
- [x] Remove both S3 buckets tied to NE account and create new ones @camposcristianeze 
- [x] Test Blueprint integration 
- [x] Deploy to Heroku
- [x] Transfer one by one all the [pages from the old repo](https://docs.google.com/spreadsheets/d/1zFOjBOt2IJFkjvrWxf-OHwQVNKWPH-YjtumqUuG9nvE/edit#gid=0)
- [x] Lazy load using Loadable
- [x] Create a new app in staging. Deploy it just using node SSR
- [x] Add first structures in contentful (redirects/images/banners) @mischacolley 

## Pages using Craft - (Remove Craft completely)
- Reports
- Be a Friend
- Stories
- Story
- Contact
- Terms of Service
- All SEO information

## Pages with more logic
- Be a Mentor (To replace)
- Positions (To replace)
- Donate (@mischacolley any ideas around this?)
